src/corehost/cli/args.cpp

```diff
@@ -89,7 +89,8 @@ bool parse_arguments(const int argc, const pal::char_t* argv[], arguments_t& arg
             {
                 trace::error(_X("Failed to locate deps file: %s"), args.deps_path.c_str());
                 return false;
-            }            
+            }      
+            args.app_dir = get_directory(args.deps_path);      
             args.app_argc = args.app_argc - 1;
```

Comment:
My concern was how would the xunit libs be loaded since that is the actual app dir but the deps file dir is assumed. @piotrpMSFT clarified offline that xunit lives inside in the NUGET_PACKAGES dir and it is part of the deps file. So it would work. All OK. LGTM.